examples/voice_chat.c

```
@@ -1,246 +1,131 @@
 
-/* Callback for received packets */
-static void radio_rx_callback(uint8_t* buffer, size_t size, int16_t rssi, int8_t snr) {
-    printf("Received packet: %zu bytes, RSSI: %d dBm, SNR: %d dB\n", size, rssi, snr);
-    
-    /* Parse the packet */
-    packet_t packet;
-    if (packet_deserialize(buffer, size, &packet) != 0) {
-        printf("Failed to parse packet\n");
-        return;
+    /* Simulate decryption by the peer */
+    uint8_t peer_key[SYMMETRIC_KEY_LENGTH];
```

Clang (14.0.6)

[warning] variable 'peer_key' is not initialized

(cppcoreguidelines-init-variables)